MTM1M3Movements.py

```diff
@@ -162,40 +168,40 @@ async def _check_position(
         if check_forces:
             # Verify there are no unintended load paths.
             self.assertAlmostEqual(
-                data.fx,
-                self.LOAD_PATH_FORCE,
-                delta=self.LOAD_PATH_TOLERANCE,
-                msg="FX out of limit",
+                data.fx * u.N,
+                self.LOAD_PATH_FORCE[0],
+                delta=self.LOAD_PATH_FORCE_TOLERANCE[0],
```

Comment:
For this and the following, I am not sure I understand the intension correctly. If you want to compare the current actuator/total force and moments with the expected values, the expected values should be a function of elevation angle at least (actually, you might also need to consider the force balance system, etc.). This means, the expectation should be a function instead of a constant.

Reply:
So far this is designed only for zenith pointing.